### DISCUSSION TITLE
GSOC 24

### DISCUSSION THREAD

**Discussion by rudranil723** [wagtail/gsoc#113] [Q&A]
I am looking forward to participating in GSOC 24 with Wagtail, as for my research and knowledge about the organization so far, I think my experience, skills, and knowledge, I will be able to contribute to the projects declared by Wagtail organization. 
I hope Wagtail participates in GSOC 24.

**Comment by HassanBahati** [comment]
hi @rudranil723 , i am super excited too and looking forward to participating in GSOC 24 with Wagtail :+1:

**Comment by thibaudcolas** [comment]
https://github.com/wagtail/gsoc#2024

**Comment by vanamaabhi2004** [comment]
Hello Wagtail Community ,
Hope you'll be into GSOC-24 and I will be looking forward to participate in GSOC and work with you.
Thank you.

**Comment by iqraakhtar09** [comment]
Hello Community I'm Iqra Akhtar. I want to contribute to wagtail and enhance my skills. I have participated in different hackathons. Now I'm feeling full of excitement and potential for this contribution. Hi mentor, any tips and tricks to crack GSOC 2024 in Wagtail?
